src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -388,6 +390,22 @@ export class ResultsViewPageStore {
                     }, false);
 
                 return anySelected;
+            },
+
+            get driverFilter() {
+                return AppConfig.serverConfig.oncoprint_custom_driver_annotation_binary_default;
+            },
+
+            set driverFilter(val:boolean) {
+                this._driverFilter = val;
+            },
+
+            get isDriverTiersSelected() {
+                return AppConfig.serverConfig.oncoprint_custom_driver_annotation_tiers_default;
+            },
+
+            set isDriverTierSelected(val:boolean) {
+                this._isDriverTiersSelected = val;
```

Comment:
I don't see the point of this `set` on line 407, you don't use it anywhere. I think we had a misunderstanding with my last comment, but I think I misunderstood so we should forget about it. Let's just get rid of this.